app/controllers/gallery_controller.rb

```diff
@@ -0,0 +1,22 @@
+# encoding: UTF-8
+class GalleryController < ApplicationController
+  load_permissions_then_authorize_resource class: false
+
+  def index
+    @albums = Album.none
+
+    if params[:year].present?
+      @year = params[:year]
+      @albums = Album.gallery(Time.new(@year))
```

Comment:
Do not use `Time.new` without zone. Use one of `Time.zone.local`, `Time.new.current`, `Time.new.in_time_zone`, `Time.new.utc`, `Time.new.getlocal`, `Time.new.iso8601`, `Time.new.jisx0301`, `Time.new.rfc3339`, `Time.new.to_i`, `Time.new.to_f` instead.